src/main/java/com/matera/cadastrocentral/authentication/AuthenticationController.java

```diff
@@ -0,0 +1,20 @@
+package com.matera.cadastrocentral.authentication;
+
+import com.matera.cadastrocentral.client.Client;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/authentication")
+public class AuthenticationController {
+
+    @Autowired
+    private AuthenticationService authenticationService;
+
+    @PatchMapping("/clients/{clientId}/password")
```

Comment:
I think the "/clients" isn't really necessary, maybe just mapping to /authentication/{clientId}/passwords or /authentication/passwords/{clientId} makes more sense. But again, since we are in a hurry keeping it the way it is might be a good ideia

Reply:
I agree that maybe the `/clients` isn't really necessary, maybe `/{clientId}/change-password`, like I said in my comment below